src/main.rs

```diff
@@ -18,12 +19,19 @@ async fn main() -> Result<()> {
     }
     match opts.subcmd {
         None => {
-            let mut force: HashSet<String> = HashSet::new();
-            for f in opts.force {
-                force.insert(f);
-            }
+            let force: HashSet<String> = HashSet::from_iter(opts.force);
+            let force_all = opts.force_all;
```

Comment:
unnecessary temporary variables?
Codes seem too long in here (

Reply:
Forget to delete it :(

Reply:
Because Rust closure does not support partial capturing yet, need to use a temporary variable. Use a closure for the possibility of extend to other flags in the future (like --force-excude).